packages/configuration-validation/package.json

```diff
@@ -1,16 +1,18 @@
 {
   "name": "@okta/configuration-validation",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
I am a bit confused with the versions  from the changelog, looks like here should be `1.1.0`